systemlink-notebook-datasource/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "rootDir": "./src",
     "baseUrl": "./src",
     "typeRoots": ["./node_modules/@types"],
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "target": "ES6"
```

Comment:
More context about this in the issue I filed: https://github.com/grafana/grafana/issues/44738

Reply:
It [looks like](https://caniuse.com/?search=es6) compiling our typescript to ES6 would only cause problems for IE11 folks. We weren't targeting any platforms that only support ES5, right? Would we want a follow-up work item to track bumping this back to ES5, or are we fine keeping this for the long haul?

Reply:
We no longer support any IE distribution so it should be fine.